### PR TITLE
Fixed #2528

### LINKED ISSUE #2528: Navigation menu labels inconsistent
### Short Summary
On the Navigation menu, the terminology is not consistent – specifically, sometimes the verb is "Go" and other times "Jump". For two others, neither verb is present.

In particular:
- For the labels "Jump to Matching Brace", "Jump to Corresponding Line (for Diff mode)", "Jump to Next Section", and "Jump to Previous Section", each "Jump" should be replaced with "Go".
- "Next Bookmark" should be changed to "Go to Next Bookmark". "Previous Bookmark" should be changed to "Go to Previous Bookmark".

### Platform Information
Komodo IDE 10.2.1 on Windows 10 Pro

### Attachments
navigation menu

### PR DESCRIPTION
**Pull request by Defman21**
fixed #2528.

I don't know if there are any IDE-specific strings, I've fixed only these which are available in the Edit version.

### CODE REVIEW

File: src/chrome/komodo/locale/en-US/keybindings/keybindings.dtd [hunk 1]
```diff
@@ -92,8 +92,8 @@
 <!ENTITY editorViMoveToTheEndOfTheSection.desc "Editor: Vi: Move to the End of the Section">
```
Comment (th3coop) [inline src/chrome/komodo/locale/en-US/keybindings/keybindings.dtd]: Lines 92, 93, and 94 seem like good candidates for this as well.

File: src/chrome/komodo/locale/en-US/komodo.dtd [hunk 2]
```diff
@@ -417,10 +417,10 @@
...
 <!ENTITY joinLines.label "Join Lines">
-<!ENTITY jumpToCorrespondingLine.label "Jump to corresponding line">
-<!ENTITY jumpToCorrespondingLineForDiffMode.label "Jump to Corresponding Line (for Diff mode)">
-<!ENTITY jumpToMatchingBrace.label "Jump to Matching Brace">
-<!ENTITY jumpToNextResult.label "Jump to next result">
+<!ENTITY jumpToCorrespondingLine.label "Go to corresponding line">
+<!ENTITY jumpToCorrespondingLineForDiffMode.label "Go to Corresponding Line (for Diff mode)">
+<!ENTITY jumpToMatchingBrace.label "Go to Matching Brace">
+<!ENTITY jumpToNextResult.label "Go to next result">
 <!ENTITY jumpToNextResult.tooltiptext "Jump To Next Result">
```
Comment (th3coop) [inline src/chrome/komodo/locale/en-US/komodo.dtd]: 424 and 425 should be changed as well.

**Comment by Defman21** [comment]
Should be good now.

**Comment by th3coop** [comment]
Thanks a lot @Defman21.

**Pull request merged**